.github/actions/upload-test-artifacts/action.yml

```diff
@@ -127,8 +127,11 @@ runs:
         # Add the run attempt, see [Artifact run attempt]
         name: test-reports-runattempt${{ github.run_attempt }}-${{ inputs.file-suffix }}.zip
         retention-days: 14
-        if-no-files-found: error
-        path: test/**/*.xml
+        # Don't want to fail the workflow here because not all workflows have csv files
+        if-no-files-found: ignore
```

Comment:
Not sure if relaxing the checking here is ok, but I think it would be fine.

Reply:
Yeah, this is even more desirable. I don't think we want to fail a workflow just because it misses some artifacts anyway